app/controllers/vendors_controller.rb

```diff
@@ -54,4 +59,9 @@ def vendor_params
     params.require(:vendor)
           .permit(:contact_name, :phone, :email, :business_name, :address)
   end
+
+  helper_method \
```

Comment:
doesn't look like this is being used.

Reply:
you may have referenced this later, but I *think* I had to put this in here so that the download link would be all set and not need changed again later. (I considered YAGNI but TBH this will probably have filters added before long and it might not be obvious that the download link would need to be changed).